.github/workflows/nightly-tests.yaml

```diff
@@ -12,19 +12,19 @@ on:
   # Day of the week ([0,6] with 0=Sunday)
   schedule:
     - cron: "0 5 * * *" # every day @ 05:00 UTC, run tests against latest main
-    - cron: "0 6 * * 1" # monday    @ 06:00 UTC, run expanded tests against v1.17.x
-    - cron: "0 7 * * 1" # monday    @ 07:00 UTC, run expanded tests against v1.16.x
-    - cron: "0 8 * * 1" # monday    @ 08:00 UTC, run expanded tests against v1.15.x
+    - cron: "0 6 * * 1" # monday    @ 06:00 UTC, run expanded tests against v1.18.x
+    - cron: "0 7 * * 1" # monday    @ 06:00 UTC, run expanded tests against v1.17.x
+    - cron: "0 8 * * 1" # monday    @ 07:00 UTC, run expanded tests against v1.16.x
   workflow_dispatch:
     inputs:
       branch:
         description: "The branch to run tests against"
         type: choice
         options:
           - main
+          - v1.18.x
           - v1.17.x
           - v1.16.x
-          - v1.15.x
```

Comment:
why don't we test on 1.15? is that still supported?

Reply:
In oss previously we only tested n-2, so I wanted to maintain that. In enterprise we still have n-3 support